packages/backend/src/graphql/schema.graphql

```diff
@@ -91,6 +91,14 @@ type Query {
     filter: PaymentFilter
   ): PaymentConnection!
 
+  "Fetch a page of accounting transfers"
+  accountingTransfers(
```

Comment:
> What are your thoughts on this? I'm hesitant to call this query "pagination" because it just takes limit - do we want to ship as is, or move it out into a separate issue and decide on the signature of this resolver separately?

That's my vote personally. Probably good not to call it pagination since its just limiting but nothing is even consuming this query ATM so I dont think we need to worry to much about improving it here.

Reply:
Yeah, I agree. Lets see how the API changes in future TigerBeetle releases. It would be cool for now to have the behaviour between the ledgers the same. I am going to resolve this comment for now. 🙇‍♂️